hack/scripts/direct_lvm.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# WARNING: THIS SCRIPT HAS MUTLIPLE PURPOSES.
+# TAKE CARE WHEN EDITING.
+
+# This is a scripted version of https://docs.docker.com/storage/storagedriver/device-mapper-driver/#configure-direct-lvm-mode-manually
+
+set -ex
+
+if [[ $(id -u) != 0 ]]; then
+  echo "Run this script as root..." >&2
+  exit 1
+fi
+
+DISK_NAME="${1:-sdb}"
+DISK_PATH="/dev/$DISK_NAME"
+POOL_NAME="${1:-flintlock}"
+PROFILE="/etc/lvm/profile/$POOL_NAME-thinpool.profile"
+CROOT=/var/lib/containerd-dev
+DIR="${CROOT}/snapshotter/devmapper"
+
+apt update
+apt install -y thin-provisioning-tools lvm2
+
```

Comment:
Be good to have some `echo` logging giving feedback as it runs.

Reply:
i have `-x` set so it is pretty loud, but yeh i can echo some things too